packages/plugin-assets-retry/src/runtime/asyncChunkRetry.ts

```diff
@@ -148,7 +147,16 @@ function ensureChunk(chunkId: string): Promise<unknown> {
       nextRetry(chunkId);
 
     if (existRetryTimes > maxRetries) {
-      error.message = `Loading chunk ${chunkId} from ${originalSrcUrl} failed after ${maxRetries} retries.`;
+      const context: AssetsRetryHookContext = {
+        times: existRetryTimes - 1,
+        domain: nextDomain,
+        url: nextRetryUrl,
+        tagName: 'script',
+      };
+      error.message = `Loading chunk ${chunkId} from ${originalSrcUrl} failed after ${maxRetries} retries. The error message of the last retry is "${error.message}"`;
```

Comment:
can we make the error message shorter, since it will be bundled to the client code.